apps/dg/components/case_table/case_table_controller.js

```diff
@@ -1019,13 +1025,153 @@ DG.CaseTableController = DG.ComponentController.extend(
         var tDataContext = this.get('dataContext'),
             tRef = tDataContext && tDataContext.getAttrRefByID( iAttrID),
             tAttrName = tRef && tRef.attribute.get('name'),
-            tAttrFormula = tRef && tRef.attribute.get('formula');
+            tAttrFormula = tRef.attribute.get('formula');
+
         DG.assert( tRef && tAttrName, "editAttributeFormula() is missing the attribute reference or attribute name" );
         // for now we use the newAttribute() method which will replace one attribute formula with another
         // if the new attribute has the same name as the old.
+
         this.editAttributeProperties({ collection: tRef.collection }, tAttrName, tAttrFormula || '' );
       },
 
+      /**
+       * Converts the formula output to raw values.
+       * @param iAttrID
+       */
+      deleteFormulaKeepValues: function (iAttrID ) {
+        var tDataContext = this.get('dataContext'),
+          tRef = tDataContext && tDataContext.getAttrRefByID(iAttrID),
+          tCollection = tRef && tDataContext.getCollectionForAttribute(tRef.attribute),
+          tAttrName = tRef && tRef.attribute.get('name'),
+          hierTableView = this.getPath('view.contentView'),
+          tFormula = '',
+          tPrevFormula = tRef && tRef.attribute.get('formula');
+
+        DG.assert( tRef && tAttrName, "deleteFormulaKeepValues() is missing the attribute reference or attribute name" );
+
+        DG.UndoHistory.execute(DG.Command.create({
+          name: "caseTable.editAttributeFormula",
+          undoString: 'DG.Undo.caseTable.editAttributeFormula',
+          redoString: 'DG.Redo.caseTable.editAttributeFormula',
+          _componentId: this.getPath('model.id'),
+          _controller: function() {
+            return DG.currDocumentController().componentControllersMap[this._componentId];
+          },
+          execute: function() {
+            var tChange = {
+                operation: 'createAttributes',
+                collection: tCollection,
+                attrPropsArray: [{ name: tAttrName, formula: tFormula }]
+              },
+              tResult = tDataContext && tDataContext.applyChange( tChange);
+            if( tResult.success) {
+              tRef.attribute.set('formula', tFormula);
+              tRef.attribute.set('deletedFormula', tPrevFormula);
+              hierTableView.updateColumnInfo();
+
+              var action = "attributeEditFormula";
+              this.log = "%@: { name: '%@', collection: '%@', formula: '%@' }".fmt(
```

Comment:
I just found that the direct call of "Delete..." / "Recover..." functions from the menu item results in "unknown action" as the log output in console. Undo / redoing, however, logs properly as `attributeEditFormula`. Not sure what's happening here...

Reply:
Usually, the DG.Command object will be created with a `log` property allong with 'name', 'undoString' and 'redoString'. Doesn't appear to be the case here.

Reply:
Usually, the DG.Command object will be created with a `log` property along with `name`, `undoString` and `redoString`. Doesn't appear to be the case here.

Reply:
It was a simple random scope problem with an unnecessary `.bind(this)`. I don't know where it came from, but logging should be fine now.